### DISCUSSION TITLE
Can we use AWS SES as smtp host in MIRTH connect

### DISCUSSION THREAD

**Discussion by gsridhar2628** [nextgenhealthcare/connect#5708] [Q&A] [answered]
Hi team,

I would like to check if we can use AWS SES as SMTP host for sending email notifications using MIRTH connect? 

We are able to send emails successfully using GMAIL smtp details, but not AWS SES details as we are getting this error when trying with SES details.

Sending the email to the following server failed : <AWS SES SMTP Host>

Can anyone please help here?

Thanks

**Comment by pacmano1** [comment, marked as answer]
Yes you can.   It's a permissions problem on the configuration of SES.

**Reply by gsridhar2628** [reply]
thanks @pacmano1 , could you please let me know how the SES configuration should be setup so that we can use it to send emails using MIRTH connect? Would be of great help to us as we are kind of blocked here.

**Reply by pacmano1** [reply]
That's an AWS config question, not a Mirth question.  Not being mean here - ask your AWS person.   There is nothing special about how Mirth calls SES vs another email relay server.

You need to consider all these things:
- Credentials to the service itself.
- Allowed To (under verified identities if so configured)
- Your sender identity (probably the "hardest part"), easy with AWS is your DNS provider.
- Port
